lib/matplotlib/mathtext.py

```diff
@@ -3320,27 +3320,28 @@ def __init__(self, output):
         """
         self._output = output.lower()
 
-    @functools.lru_cache(50)
-    def parse(self, s, dpi = 72, prop = None):
+    def parse(self, s, dpi=72, prop=None):
         """
-        Parse the given math expression *s* at the given *dpi*.  If
-        *prop* is provided, it is a
-        :class:`~matplotlib.font_manager.FontProperties` object
-        specifying the "default" font to use in the math expression,
-        used for all non-math text.
+        Parse the given math expression *s* at the given *dpi*.  If *prop* is
+        provided, it is a `.FontProperties` object specifying the "default"
+        font to use in the math expression, used for all non-math text.
 
         The results are cached, so multiple calls to :meth:`parse`
         with the same expression should be fast.
         """
+        return self._parse_cached(
```

Comment:
Please add a comment that we have to defer to a seprate function and parse the rcParams in order to get poper caching behavior.

Reply:
done